### ISSUE TITLE
Get pinned tags from database

### ISSUE THREAD

**Issue by nberlee** [g3rv4/GetMoarFediverse#25]
Tags can either be followed (flooding the timeline) or pinned in the advanced webinterface, creating an extra lane.

Pinned tags are not saved in the `tags` table, but in a json object in table `web_settings`

It would be great if GetMoarFediverse could iterate over this table to get all pinned tags.
They are located in in data->'columns'-> Array item where id='HASHTAG' -> params -> id

**Comment by nberlee** [comment]
The query:
```sql
SELECT col->'params'->>'id'
FROM web_settings, json_array_elements(data->'columns') col
WHERE col->>'id' = 'HASHTAG'
AND col->'params'->>'id' IS NOT NULL
;
```

**Comment by nberlee** [comment]
I will create a PR

**Comment by g3rv4** [comment]
appreciate it! please make it optional though... I'd make it so that if you enter the connection string, you can decide what you want to import

**Comment by g3rv4** [comment]
I'm working on refactoring a bit of this mess, but I'm happy to pick up your PR and adjusting it

**Comment by g3rv4** [comment]
Thank you for the PR! This is now included on [v1.0.43](https://github.com/g3rv4/GetMoarFediverse/releases/tag/v1.0.43)
